modules/build/src/test/scala/scala/build/tests/BuildTests.scala

```diff
@@ -399,7 +421,7 @@ class BuildTests extends munit.FunSuite {
           |}
           |""".stripMargin,
       os.rel / "Ignored.scala" ->
-        """using target scala == 2.12
+        """// @using target.scala.== "2.12"
```

Comment:
I had no idea that we support `// @using` syntax

Reply:
Is `// using lib ...` supported?

Reply:
Yes

Reply:
@alexarchambault @tpasternak @lwronski I have my doubts now if we should support `// @using ...` sytnax. For me e.g. `// @using scala 3` should be a commented using directive

Reply:
If we agree that we do not want `// @ syntax` then we should have a test to make sure it is not recoginized